core/src/autogluon/core/_setup_utils.py

```diff
@@ -17,10 +17,10 @@
 DEPENDENT_PACKAGES = {
     # note: if python 3.7 is used, the open CVEs are present: CVE-2021-41496 | CVE-2021-34141; fixes are available in 1.22.x, but python 3.8 only
     'numpy': '>=1.21,<1.23',
-    'pandas': '>=1.2.5,<1.4',  # Capped version of pandas to 1.4.0 because of issue: https://github.com/pandas-dev/pandas/issues/45603
+    'pandas': '>=1.2.5,!=1.4.0',
```

Comment:
Can we add a cap of `<2`? Note that generally we version cap to minor versions prior to our releases because I don't trust our dependencies to not add breaking changes in minor releases (unfortunately this is from experience).

I'm ok with keeping the `<6` for psutil since it is such a minor package. As for pandas `<2` is fine for now but I'll probably version cap it again prior to v0.5 release. (Even if it isn't breaking changes, they can still introduce deprecation warnings that spam users)

Reply:
```suggestion
    'pandas': '>=1.2.5,!=1.4.0,<1.5',
```
I totally feel you 😅 better safe than sorry!